Tasks/Common/nuget-task-common/module.json

```diff
@@ -1,19 +1,30 @@
 {
     "messages": {
-        "NGCommon_IgnoringNuGetExtensionsPath": "Detected NuGet extensions loader path (NUGET_EXTENSIONS_PATH environment variable). Extensions are ignored when using the built-in NuGet client",
-        "NGCommon_DetectedNuGetExtensionsPath": "Detected NuGet extensions loader path (NUGET_EXTENSIONS_PATH environment variable): %s",
-        "NGCommon_UnableToFindTool": "Unable to find tool %s",
+        "Error_ApiKeyNotSupported": "DotNetCore currently does not support using an encrypted Api Key.",
```

Comment:
Treat this list as a master list of all nuget-task-common resources. Should be copied as a block into any task using the nuget-task-common package.